functions/http/1.1/index.js

```diff
@@ -0,0 +1,58 @@
+import Liquid from './liquid.min';
+
+const engine = new Liquid();
+
+const parseHeaders = (headers) =>
+  headers.reduce((acc, { key, value }) => ({ ...acc, [key]: value }), {});
+
+const parseQueryParameters = (queryParameters) =>
+  queryParameters
+    .map(({ key, value }, index) => {
+      const paramKey = index === 0 ? `?${key}` : key;
+      return `${paramKey}=${encodeURIComponent(value)}`;
+    })
+    .join('&');
+
+const parseLiquid = async (body, bodyParameters) =>
+  engine.parseAndRenderSync(
+    body,
+    bodyParameters.reduce((parameter, { key, value }) => {
+      // eslint-disable-next-line no-param-reassign
+      parameter[key] = value;
+      return parameter;
+    }, {}),
+  );
+
+const generateUrl = (url, protocol, queryParameters) =>
+  `${protocol}://${url}${parseQueryParameters(queryParameters)}`;
+
+const http = async ({
+  url,
+  method,
+  body,
+  headers = [],
+  protocol,
+  queryParameters = [],
+  bodyParameters,
```

Comment:
Is there a reason for not giving `bodyParameters` and `urlParameters` a default value?